Source/Async.swift

```diff
@@ -580,8 +460,11 @@ public struct Apply {
          - iterations: How many times the block should be run. Index provided to block goes from `0..<iterations`
          - block: The block that is to be passed to be run on a .
      */
-    public static func userInteractive(iterations: Int, block: Int -> ()) {
-        dispatch_apply(iterations, GCD.userInteractiveQueue(), block)
+    public static func userInteractive(_ iterations: Int, block: (Int) -> ()) {
+//        let dispatchWorkItem = DispatchWorkItem(qos: .userInteractive, block: block)
+//        DispatchQueue.concurrentPerform(iterations: iterations, execute: dispatchWorkItem)
+        // TODO: Find workable API for apply on non-main queue
+        DispatchQueue.concurrentPerform(iterations: iterations, execute: block)
```

Comment:
I haven't been able to figure out how to use the new `Dispatch.concurrentPerform(iteration:, execute:)` on thread other than main.